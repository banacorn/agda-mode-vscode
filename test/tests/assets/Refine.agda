-- from issue #158
module Refine where

postulate
  A : Set

record Foo : Set where
  field
    very-long-field-name-1 : A
    very-long-field-name-2 : A
    very-long-field-name-3 : A

foo : Foo
foo = {!   !}

bar : Foo
bar = {!   !}