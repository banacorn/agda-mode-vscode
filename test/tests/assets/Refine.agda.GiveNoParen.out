-- https://github.com/banacorn/agda-mode-vscode/issues/158
module Refine where

postulate
  A : Set

record Foo : Set where
  field
    very-long-field-name-1 : A
    very-long-field-name-2 : A
    very-long-field-name-3 : A

foo : Foo
foo = {!   !}

open import Lib

fst : Bool → Bool → Bool
fst a b = a

bar : Bool
bar = fst true {!   !}